### ISSUE TITLE
Create one more Home Card

### ISSUE THREAD

**Issue by DamyanBG** [DamyanBG/real-estate-react-app#383]
Add one more home card

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/b3bc439a-cc89-476a-b1d3-cea74883fafd)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/88bc6327-e4ef-48c5-b2b8-00311f769d34)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/7a8e6beb-48dd-4aba-9b8f-381a89eaad57)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/b80cb92d-52c1-4dca-9314-47b2f7ffeb30)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/98905799-f611-4876-bc46-9d236102d9a6)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/1d408124-c629-46a6-9d6e-c97054b7fde4)

**Comment by DamyanBG** [comment]
![image](https://github.com/DamyanBG/real-estate-react-app/assets/93829069/c2ee6e86-bcf9-4e68-abf0-ccb24aaa8cca)

**Comment by tihomirtx88** [comment]
https://prnt.sc/CnSgH1m4ZDon This sugesting look like will cover most of styles on the webside like design i will start working on it .

**Comment by DamyanBG** [comment]
Yes, I liked this also, but I think we can implement more, to have some portfolio.
